packages/client-web-kit-theme/assets/css/styles/login.css

```
@@ -0,0 +1,31 @@
+ * property with a plain Tailwind utility on the element.
+ */
+@layer components {
+    .a-login-provider-item {
+        min-width: 0;
+        max-width: 100%;
+    }
+
+    .a-login-provider-box {
+        min-width: var(--authup-login-provider-min-width);
+        max-width: 100%;
+        word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)